functions.php

```diff
@@ -1024,4 +1024,19 @@ function display_gmucf_template($template) {
 }
 add_action('template_redirect', 'gmucf_template_redirect', 1);
 
+function displaySocialShare( $permalink, $title ) {
```

Comment:
Style guide stuff: lowercase + underscore-separated function names are preferred for php functions (`display_social_share`)